libs/wingcompiler/src/compile.ts

```diff
@@ -164,8 +164,7 @@ export async function compile(entrypoint: string, options: CompileOptions): Prom
   if (!existsSync(synthDir)) {
     await fs.mkdir(workDir, { recursive: true });
   }
-
-  let preflightEntrypoint = await compileForPreflight({
+  let compileForPreflightResult = await compileForPreflight({
```

Comment:
nit: Any reason these are not `const`?

Reply:
😁 Yea I cant every remember what language Im in. I think const is fine here